scss/tech.scss

```diff
@@ -0,0 +1,47 @@
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  height: 350px;
+  width: 260px;
+
+  &::before {
+    border: 2px solid $green;
+    border-radius: 10px;
+    content: '';
+    height: 100%;
+    left: -15px;
+    position: relative;
```

Comment:
this should be absolute positioned, while the position of the content of the tech (title, logo, etc) should not be altered